test/priorityQueue.js

```diff
@@ -31,26 +31,25 @@ describe('priorityQueue', () => {
             expect(q.length()).to.equal(0);
             call_order.push('callback ' + 3);
         });
-        q.push(4, 2.9, (err, arg) => {
```

Comment:
The expected lengths change, due to pushing this array, correct?

Reply:
Yes sir, we don't test pushing an array in any of the other `priorityQueue` tests, so I thought it would be useful to add.